{
	"@context": "https://schema.org",
	"@type": "Article",
	"publisher": {
		"@type": "Organization",
		"name": "The Cloudflare Blog",
		"logo": {
			"@type": "ImageObject",
			"url": "https://blog-cloudflare-com-assets.storage.googleapis.com/2019/06/logo-cloudflare-dark-1.svg",
			"width": 109,
			"height": 40.5
		}
	},
	"author": {
		"@type": "Person",
		"name": "Rushil Shah",
		"image": {
			"@type": "ImageObject",
			"url": "http://blog.cloudflare.com/content/images/2021/12/8B764ABE-D6C2-47FD-AD51-4C9E762BCD00.jpeg",
			"width": 1170,
			"height": 1064
		},
		"url": "https://blog.cloudflare.com/author/rushil/",
		"sameAs": []
	},
	"headline": "How Cloudflare security responded to Log4j 2 vulnerability",
	"url": "https://blog.cloudflare.com/how-cloudflare-security-responded-to-log4j2-vulnerability/",
	"datePublished": "2021-12-10T23:39:00.000Z",
	"dateModified": "2021-12-17T13:30:20.000Z",
	"image": {
		"@type": "ImageObject",
		"url": "http://blog.cloudflare.com/content/images/2021/12/4A4B98A3-9C9E-4CDB-B216-578B044A36B3.jpeg",
		"width": 1170,
		"height": 633
	},
	"keywords": "Vulnerabilities, Security, Log4J, Log4Shell",
	"description": "Yesterday, December 9, 2021, when a serious vulnerability in the popular Java-based logging package log4j was publicly disclosed, our security teams jumped into action to help respond to the first question and answer the second question. This post explores the second.",
	"mainEntityOfPage": {
		"@type": "WebPage",
		"@id": "http://blog.cloudflare.com/"
	}
}